Frends.SFTP.WriteFile/Frends.SFTP.WriteFile/Frends.SFTP.WriteFile.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.ComponentModel;
+using Renci.SshNet;
+
+#pragma warning disable 1591
+
+namespace Frends.SFTP.WriteFile
+{
+    public class SFTP
+    {
+
+        /// <summary>
+        /// Writes a file with SFTP connection.
+        /// Documentation: https://github.com/FrendsPlatform/Frends.SFTP.WriteFile
+        /// </summary>
+        /// <param name="connection">Transfer connection parameters</param>
+        /// <param name="source">Source file location</param>
+        /// <param name="destination">Destination directory location</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>Result object </returns>
+        public static Result WriteFile([PropertyTab] Source source, [PropertyTab] Destination destination, [PropertyTab] Connection connection, CancellationToken cancellationToken)
+        {
+            return new SFTP().WriteFileInternal(connection, source, destination, cancellationToken);
+        }
+
+        internal Result WriteFileInternal(Connection connection, Source source, Destination destination, CancellationToken cancellationToken)
+        {
+            // Establish connectionInfo with connection parameters
+            var connectionInfo = GetConnectionInfo(connection);
+            Result result = null;
+            
+            try
+            {
+                using (var client = new SftpClient(connectionInfo))
+                {
+                    client.Connect();
+                    client.ChangeDirectory(destination.Directory);
+                    client.BufferSize = 1024;
+                    result = TransferSingleFile(client, source, destination);
+                    client.Disconnect();
+
+                }
+            }
+            catch (Renci.SshNet.Common.SshConnectionException ex)
+            {
+                throw new Exception($"Error when establishing connection to the Server: {ex.Message}", ex);
+            }
+            catch (System.Net.Sockets.SocketException ex)
+            {
+                throw new Exception($"Unable to establish the socket: No such host is known.", ex);
+            }
+            catch (Renci.SshNet.Common.SshAuthenticationException ex)
```

Comment:
You can remove namespace definition after it is added to the beginning of the file.

Reply:
Fixed